lua/nvim-tree.lua

```diff
@@ -621,6 +621,7 @@ local DEFAULT_OPTS = { -- BEGIN_DEFAULT_OPTS
       resize_window = true,
       window_picker = {
         enable = true,
+        picker = "default",
```

Comment:
I've changed this to a string/function to leave the door open for other methods in the future.